scripts/govuk_app_owners.coffee

```diff
@@ -0,0 +1,12 @@
+# This script teaches Hubot to answer questions like "Who owns Collections Publisher?"
+# and "Who owns publishing-api?"
+module.exports = (robot) ->
+  robot.hear /who owns (.*)\?/i, (res) ->
+    # Best guess of the intended application.
+    application = res.match[1].replace(' ', '-').toLowerCase()
```

Comment:
Nope, only govuk_need_api, which is going to be deprecated:

https://docs.publishing.service.gov.uk/apps.html